locales/en.default.json

```diff
@@ -605,7 +720,9 @@
             "only_left": "Only {{ count }} left!",
             "quantity": "Quantity",
             "add_to_cart": "Add to Cart",
-            "details_tab": "Product details"
+            "details_tab": "Product details",
+            "inc": "Inc VAT",
```

Comment:
`inc` → `vat` (it's tax name in most countries, while inc is just short of include)

Reply:
where is updates?